megamek/src/megamek/server/Server.java

```diff
@@ -6297,36 +6057,36 @@ private void doVehicleFlipDamage(Tank entity, int damage, boolean startRight, in
 
         int index = flipCount % 4;
         // If there is no turret, we do side-side-bottom
-        if (((Tank)entity).hasNoTurret()) {
+        if (entity.hasNoTurret()) {
             index = flipCount % 3;
             if (index > 0) {
                 index++;
             }
         }
         switch (index) {
-        case 0:
-            hit = new HitData(startRight? Tank.LOC_RIGHT : Tank.LOC_LEFT);
-            break;
-        case 1:
-            hit = new HitData(Tank.LOC_TURRET);
-        case 2:
-            hit = new HitData(startRight? Tank.LOC_LEFT : Tank.LOC_RIGHT);
-            break;
-        default:
-            hit = null; //Motive damage instead
+            case 0:
+                hit = new HitData(startRight ? Tank.LOC_RIGHT : Tank.LOC_LEFT);
+                break;
+            case 1:
+                hit = new HitData(Tank.LOC_TURRET);
+                break; //TODO : Windchild Ensure this change is correct
```

Comment:
I'm pretty sure this is a bug, but wanted to check that the missing break wasn't on purpose.

Reply:
Yes, that's a bug.